lib/engines/aws.rb

```diff
@@ -92,5 +94,87 @@ def translate(input_text, src_lang, target_lang)
       :source_language_code => "#{src_lang}", :target_language_code => "#{target_lang}"})
     response.translated_text
   end
+
+  def transcribe_uri(video_uri, lang_code, bucket_name)
+    # output name of the transcribe file will be based on the Job name
+    # So better create our own job name
+    # They have to give the bucket name. Since we dont have access to their buckets
+    # Both input and output files
+
+
+    current_time = Time.now.strftime("%Y-%m-%d-%H-%M-%S")
+    job_name = "transcribe-#{current_time}"
+
+    #So output filename will be job_name.json
+    resp = @transcribe_service.start_transcription_job({
+        transcription_job_name: job_name,
+        language_code: lang_code,
+        media: {
+            media_file_uri: video_uri
+        },
+        output_bucket_name: bucket_name
```

Comment:
1. Should we have a root directory specified instead of storing stuff at bucket level ?
2. If the user doesn't offer an output bucket, I believe Transcribe gives back a signed url to the job output. If that's true can we support that mode as well ?

Reply:
1. According the Transcribe, the file is stored in bucket level only.
2. Transcribe doesnt give signed url to the job output. It stores the output at bucket level in s3.